hardware/run_all_sims.py

```diff
@@ -0,0 +1,74 @@
+from subprocess import run
+from os import remove, makedirs, path
+from shutil import rmtree
+from typing import Tuple
+
+RESULT_DIR = 'test_results'
+
+def run_bench(name, cleanup, success_cond):
```

Comment:
Could you add type annotations for the arguments and return value of this function

Reply:
Sure! Added :)